BusBoard.ConsoleApp/TflApi.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using RestSharp;
+
+namespace BusBoard.ConsoleApp
+{
+    public class TflApi
+    {
+        public IEnumerable<BusStop> GetTwoClosestBusStopsToPostcode(string postcodeString)
+        {
+            var requestUrl = "http://api.postcodes.io";
+            var client = new RestClient(requestUrl);
+            var request = new RestRequest($"postcodes/{postcodeString}", DataFormat.Json);
+            var response = client.Execute(request);
+            var postcodeApiResponse = JsonConvert.DeserializeObject<PostcodeApiResponse>(response.Content);
```

Comment:
I think you should be able to use RestSharp in a similar way to in `GetListOfArrivalPredictionsForStopPoint`, and avoid the need to use JsonConvert here

Reply:
I tried to and I don't think it worked, though I was rushing a bit, I'll try again